docs/firststept.rst

```diff
@@ -57,9 +66,11 @@ When the user presses the Subscriber button a new window appear to allow the use
 
 This menu is very similar to the Publication menu but the user cannot change the color and size, and it has additional elements:
 
-- **Time Based Filter:** This value can be used by the user to only specify the minimum amount of time (in milliseconds) that the subscriber wants between updates. 
+- **Time Based Filter:** This value can be used by the user to only specify the minimum amount of time (in milliseconds) that the subscriber wants between updates.
+
+- **Content Based Filter:** This filter draws a rectangle in the shapes window. Only the instances that are included in this rectangle are accepted. The rest of them are ignored. The user can dynamically resize and move this content filter.
 
```

Comment:
Sorry I missed this in my previous review, but `Deadline` and `Lifespan` should also be added here as they also apply to the subscriber side